react/features/app/components/AbstractApp.js

```diff
@@ -201,7 +228,10 @@ export class AbstractApp extends Component {
             return (
                 <I18nextProvider i18n = { i18next }>
                     <Provider store = { this._getStore() }>
-                        { this._createElement(component) }
+                        <AppRoot>
```

Comment:
I'll replace with a Fragment as soon as we update React: https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html